assets/js/app.js

```diff
@@ -320,6 +349,47 @@ app.controller('ClassCreateCtrl', function($scope, $rootScope, $http, $location,
   }
 })
 
+app.controller('ProjectCtrl', function($scope, $rootScope, $http, $routeParams, Consts) {
+  var class_name = $routeParams.class_name
+  var project_name = $routeParams.project_name
+
+  $rootScope.root = {
+    title: 'Project ' + project_name + ' | GitSubmit'
+  }
+
+  $http({
+    method: 'GET',
+    url: Consts.API_SERVER + '/classes/' + class_name + "/projects/" + project_name + "/"
+  }).then(function(res) {
+    $scope.project_description = res.data.project.description
```

Comment:
you can just export `res.data.project` to `$scope.project` then in the html markup do `{{ project.description }}`